commands/tocfl/tocfl.py

```diff
@@ -56,10 +57,16 @@ async def tocfl_rand(
 def _create_word_embed(
     word: str, level: int, part_of_speech: str, pinyin: str
 ):
+    try:
+        chewing = to_chewing(pinyin)
+    except AssertionError:
+        chewing = "ERROR OCCURRED"

```

Comment:
The ERROR OCCURRED should only be visible to devs and not users, so you should set `chewing = None` and then print the error to console.

So replace this with something like:
```py
except AssertionError as e:
  print("error occurred during zhuyin conversion:", e)
  chewing = None
```